waspc/e2e-test/Main.hs

```diff
@@ -0,0 +1,43 @@
+import Control.Monad (unless)
+import System.Directory (getCurrentDirectory)
+import System.FilePath (joinPath, takeFileName, (</>))
+import System.Info
+import Test.Tasty (TestTree, defaultMain, testGroup)
+import Util
+  ( newAndCompileAndMigrateCommand,
+    newAndCompileCommand,
+    newCommand,
+    runGoldenTest,
+  )
+
+-- TODO: Put `cwd` into Reader MonadT?
+
+main :: IO ()
+main = do
+  -- Skip Windows in CI for now
+  unless (os == "mingw32") $ do
```

Comment:
Ah I see what you mean! Makes sense!

So on one hand we have e2e tests being silently skipped on Windows, which doesn't sound great, but on the other hand, we have `stack test` failing on Windows, which is also not fun.
At the end both cases are not really important because I doubt much/any development will be done directly on Windows.
Ok, whatever you choose to do is fine with me!

Reply:
For now I put it into a message to the user that should hopefully inform the intent without a comment:

```haskell
if os == "mingw32"
    then putStrLn "Skipping end-to-end tests on Windows due to tests using *nix-only commands"
```

I'l chew on if a comment about erroring out vs not (and `stack test` implications) makes sense to add 👍🏻

Reply:
I think this is already great!